api/browser_context.go

```diff
@@ -41,15 +41,16 @@ type BrowserContext interface {
 //
 // https://datatracker.ietf.org/doc/html/rfc6265.
 type Cookie struct {
-	Name     string         `json:"name"`          // Cookie name.
-	Value    string         `json:"value"`         // Cookie value.
-	Domain   string         `json:"domain"`        // Cookie domain.
-	Path     string         `json:"path"`          // Cookie path.
-	Expires  int64          `json:"expires"`       // Cookie expiration date as the number of seconds since the UNIX epoch.
-	HTTPOnly bool           `json:"httpOnly"`      // True if cookie is http-only.
-	Secure   bool           `json:"secure"`        // True if cookie is secure.
-	SameSite CookieSameSite `json:"sameSite"`      // Cookie SameSite type.
-	URL      string         `json:"url,omitempty"` // Cookie URL.
+	Name     string         `json:"name"`                   // Cookie name.
+	Value    string         `json:"value"`                  // Cookie value.
+	Domain   string         `json:"domain"`                 // Cookie domain.
+	Path     string         `json:"path"`                   // Cookie path.
```

Comment:
Just a suggestion, but also happy to leave it as is.

Just for consistency, would it be a good idea to add the `js` struct tags to all the fields?

Reply:
Sure thing, ac67ec1 👍

Reply:
Thanks! 🎉